payments-core/src/main/java/com/stripe/android/model/CardParams.kt

```diff
@@ -16,41 +16,36 @@ data class CardParams internal constructor(
     private val loggingTokens: Set<String> = emptySet(),
 
     /**
-     * [card.number](https://stripe.com/docs/api/tokens/create_card#create_card_token-card-number)
-     *
-     * Required
-     *
      * The card number, as a string without any separators.
+     *
+     * [card.number](https://stripe.com/docs/api/tokens/create_card#create_card_token-card-number)
      */
-    internal var number: String,
+    @property:DelicateCardDetailsApi
+    val number: String,
 
     /**
-     * [card.exp_month](https://stripe.com/docs/api/tokens/create_card#create_card_token-card-exp_month)
-     *
-     * Required
-     *
      * Two-digit number representing the card's expiration month.
+     *
+     * [card.exp_month](https://stripe.com/docs/api/tokens/create_card#create_card_token-card-exp_month)
      */
-    internal var expMonth: Int,
+    @property:DelicateCardDetailsApi
+    val expMonth: Int,
 
     /**
-     * [card.exp_year](https://stripe.com/docs/api/tokens/create_card#create_card_token-card-exp_year)
-     *
-     * Required
-     *
      * Two- or four-digit number representing the card's expiration year.
+     *
+     * [card.exp_year](https://stripe.com/docs/api/tokens/create_card#create_card_token-card-exp_year)
      */
-    internal var expYear: Int,
+    @property:DelicateCardDetailsApi
+    val expYear: Int,
 
     /**
-     * [card.cvc](https://stripe.com/docs/api/tokens/create_card#create_card_token-card-cvc)
-     *
-     * Usually required
+     * Card security code.
      *
-     * Card security code. Highly recommended to always include this value, but it's required only
-     * for accounts based in European countries.
+     * [card.cvc](https://stripe.com/docs/api/tokens/create_card#create_card_token-card-cvc)
      */
-    internal var cvc: String? = null,
```

Comment:
Out of curiosity, why were these vars before?

Reply:
🤷